src/System.Xml.XDocument/ref/System.Xml.XDocument.cs

```diff
@@ -121,6 +121,7 @@ public partial class XCData : System.Xml.Linq.XText
         public XCData(System.Xml.Linq.XCData other) : base(default(string)) { }
         public override System.Xml.XmlNodeType NodeType { get { throw null; } }
         public override void WriteTo(System.Xml.XmlWriter writer) { }
+        public override System.Threading.Tasks.Task WriteToAsync(System.Xml.XmlWriter writer, System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
I believe all of these new APIs in the contracts need to be surrounded by `#if netcoreapp11`. See some of the other PRs that contribute to #2869 like #12600 and #13828.